migrations/versions/407e74de5553_adding_user_table.py

```diff
@@ -8,14 +8,13 @@
 
 # revision identifiers, used by Alembic.
 revision = '407e74de5553'
-down_revision = '3a5eba38e4e8'
+down_revision = '12f9dcc2e36c'
```

Comment:
These alembic versions get really messed up with git branches. It's really hard to know the order of application as the DB only has the most recent - which depending on which code you've branched from is not the best measure - not the application history. Maybe we need to manage these outside of the application?